charts/sscs-tribunals-api/values.ccd.preview.template.yaml

```diff
@@ -212,8 +212,11 @@ ccd:
     clusterName: "es"
     replicas: 1
   logstash:
-    image:
-      tag: ccd-sscs-logstash-latest
+    image:logstash:
```

Comment:
Why did we change this ?

Reply:
I will change this back as I wasn't aware this had been changed.